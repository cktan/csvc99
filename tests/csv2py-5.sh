# Test Case : CSV with Various Special Characters
../csv2py -e '\' in/csv2py-5.csv 
