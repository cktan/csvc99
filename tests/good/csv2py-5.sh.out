[
	['name','info'],
	['John','He said, "Hello, world!"'],
	['Jane','Path: C:\\Users\\Jane']
]

